spec.html

```diff
@@ -45393,7 +45393,6 @@ <h1>
           1. Assert: _genContext_ is the execution context of a generator.
           1. Let _generator_ be the value of the Generator component of _genContext_.
           1. Assert: GetGeneratorKind() is ~async~.
-          1. Set _value_ to ? Await(_value_).
```

Comment:
I'm implementing this in Babel, and I need help to understand this change: is it observable when yielding promises?

```js
async function* fn() {
  yield Promise.resolve(0);
}

fn().next().then(console.log); // { value: 0, done: false } or { value: Promise.resolve(0), done: false }?
```

Reply:
:facepalm: The await has just been moved, not deleted.